how-to-guides/e2e-tracking/notebooks/e2e_tracking.ipynb

```diff
@@ -0,0 +1,749 @@
+    "        \"sys/id\"\n",
+    "    ].values[0]\n",
+    "    champion_model_score = model_versions_df[model_versions_df[\"sys/stage\"] == \"production\"][\n",
+    "        \"training/score\"\n",
+    "    ].values[0]\n",
+    "    print(f\"Champion model ID: {champion_model} and score: {champion_model_score}\")\n",
+    "    NO_CHAMPION = False\n",
+    "except IndexError:\n",
+    "    print(\"❌ No model found in production\")\n",
+    "    NO_CHAMPION = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "staged_models = model_versions_df[model_versions_df[\"sys/stage\"] == \"staging\"]\n",
+    "challenger_model_score = min(staged_models[\"training/score\"])\n",
+    "challenger_model_id = staged_models[staged_models[\"training/score\"] == challenger_model_score][\n",
+    "    \"sys/id\"\n",
+    "].values[0]\n",
+    "\n",
+    "print(f\"Challenger model ID: {challenger_model_id} and score: {challenger_model_score}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Promote challenger to champion if score is better"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if NO_CHAMPION:\n",
+    "    print(f\"Promoting {challenger_model_id} to Production\")\n",
+    "    with neptune.init_model_version(with_id=challenger_model_id) as challenger_model:\n",
+    "        challenger_model.change_stage(\"production\")\n",
+    "\n",
+    "elif challenger_model_score < champion_model_score:\n",
+    "    print(\"Challenger is better than champion\")\n",
+    "\n",
+    "    print(f\"Archiving champion model {champion_model}\")\n",
+    "    with neptune.init_model_version(with_id=champion_model) as champion_model:\n",
+    "        champion_model.change_stage(\"archived\")\n",
+    "\n",
+    "    print(f\"Promoting {challenger_model_id} to Production\")\n",
+    "    with neptune.init_model_version(with_id=challenger_model_id) as challenger_model:\n",
+    "        challenger_model.change_stage(\"production\")\n",
+    "\n",
+    "else:\n",
+    "    print(\"Champion model is better than challenger\")\n",
+    "    print(f\"Archiving challenger model {challenger_model_id}\")\n",
+    "    with neptune.init_model_version(with_id=challenger_model_id) as challenger_model:\n",
+    "        challenger_model.change_stage(\"archived\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Monitor model in production\n",
+    "In this section, you will:\n",
+    "1. Download the model binary from the model registry to make predictions in production\n",
+    "2. Use [EvidentlyAI](https://www.evidentlyai.com/) to monitor your model in production.\n",
+    "\n",
+    "You will use a modified version of the Evidently tutorial available [here](https://docs.evidentlyai.com/get-started/tutorial)."
```

Comment:
Tone

```suggestion
    "In this section, we'll:\n",
    "1. Download the model binary from the model registry to make predictions in production\n",
    "2. Use [EvidentlyAI](https://www.evidentlyai.com/) to monitor the model in production.\n",
    "\n",
    "We'll use a modified version of a tutorial from the [Evidently documentation](https://docs.evidentlyai.com/get-started/tutorial)."
```

Reply:
[Not necessarily](https://github.com/neptune-ai/examples/pull/298#discussion_r1512786059)